runtime/commonMain/src/kotlinx/serialization/internal/NullableSerializer.kt

```diff
@@ -56,6 +56,7 @@ public class NullableSerializer<T : Any>(private val serializer: KSerializer<T>)
     }
 
     private class SerialDescriptorForNullable(private val original: SerialDescriptor): SerialDescriptor by original {
+        override val serialName: String = original.serialName + ".nullable"
```

Comment:
Because it's human-readable and saving a few characters is not the goal here.

>Shouldn't we try to wrap serializer into NullableSerializer every time we see ? at the end of serial name?

I am not sure I understand. Why should we do anything with serializers just based on the descriptor name